### PR TITLE
Add a blog post about research projects using Chainer

### PR DESCRIPTION
**Pull request by mitmul**

### CODE REVIEW

File: _posts/2016-12-01-Research-projects-using-Chainer.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Research projects using Chainer
+layout: post
+categories: General
+---
+
+Recently we found some great research projects that are using Chainer for their algorithm implementations and experiments. We searched for such publicly available projects on arXiv and summarized them here as a table that lists papers up with their URL links: [Research projects using Chainer](https://github.com/pfnet/chainer/wiki/Research-projects-using-Chainer).
```
Comment (bkvogel) [inline _posts/2016-12-01-Research-projects-using-Chainer.md]: Wording: I suggest:

that lists papers along with their URL links

Comment (mitmul) [inline _posts/2016-12-01-Research-projects-using-Chainer.md]: Thanks! Fixed it.

File: _posts/2016-12-01-Research-projects-using-Chainer.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+As for the conference the papers submitted, [International Conference on Learning Representations (ICLR)](http://www.iclr.cc) is the dominant conference that has papers using Chainer. ICLR is one of the top conferences, targets representation learning and actually focuses on Deep Learning, so that many papers tend to propose new architectures and those sometimes need very flexible framework to be implemented. These really motivate us to keep Chainer flexible and enabled to accelerate try-and-error cycles in research.
```
Comment (bkvogel) [inline _posts/2016-12-01-Research-projects-using-Chainer.md]: Wording: I suggest to change

As for the conference the papers submitted,

to:

As for the conference of the submitted papers,

Comment (mitmul) [inline _posts/2016-12-01-Research-projects-using-Chainer.md]: Thanks! Fixed it too.

**Pull request merged**